markdown/design-to-code-workshop/design-to-code-workshop.md

```diff
@@ -236,8 +237,8 @@ Fill in the fields as follows:
             ]
         }
     }
-}
-```  
+}       
```

Comment:
Did the extra spaces actually need to be added here or was this on accident?

Reply:
Yea it had to be added to represent an ending of a code block since the NodeCode parser looks for `\n`

Reply:
thanks.